src/views/account/Chat/Modals/Chat.tsx

```diff
@@ -1,30 +1,30 @@
-import React, { useEffect, useRef, useState} from "react";
-import { Image, ActivityIndicator, FlatList, ImageBackground, Platform, StyleSheet, Text, TextInput, TouchableOpacity, View, KeyboardAvoidingView,} from "react-native";
-import { useTheme} from "@react-navigation/native";
+import React, { useEffect, useRef, useState } from "react";
+import { Image, ActivityIndicator, FlatList, ImageBackground, Platform, StyleSheet, Text, TextInput, TouchableOpacity, View, KeyboardAvoidingView, } from "react-native";
+import { useTheme } from "@react-navigation/native";
 
-import type {Screen} from "@/router/helpers/types";
-import { NativeText,} from "@/components/Global/NativeComponents";
-import {useCurrentAccount} from "@/stores/account";
-import type {ChatMessage, ChatRecipient} from "@/services/shared/Chat";
-import {ChevronLeft, File, Link, Send} from "lucide-react-native";
+import type { Screen } from "@/router/helpers/types";
+import { NativeText, } from "@/components/Global/NativeComponents";
+import { useCurrentAccount } from "@/stores/account";
+import type { ChatMessage, ChatRecipient } from "@/services/shared/Chat";
+import { ChevronLeft, Send } from "lucide-react-native";
 import parse_initials from "@/utils/format/format_pronote_initials";
 import InitialIndicator from "@/components/News/InitialIndicator";
-import {PapillonModernHeader} from "@/components/Global/PapillonModernHeader";
-import {useSafeAreaInsets} from "react-native-safe-area-context";
+import { PapillonModernHeader } from "@/components/Global/PapillonModernHeader";
+import { useSafeAreaInsets } from "react-native-safe-area-context";
 import HTMLView from "react-native-htmlview";
-import Reanimated, {FadeIn, FadeInDown, FadeOut} from "react-native-reanimated";
-import {AccountService} from "@/stores/account/types";
+import Reanimated, { FadeIn, FadeInDown, FadeOut } from "react-native-reanimated";
+import { AccountService } from "@/stores/account/types";
 import * as WebBrowser from "expo-web-browser";
-import {WebBrowserPresentationStyle} from "expo-web-browser";
+import { WebBrowserPresentationStyle } from "expo-web-browser";
 import getAndOpenFile from "@/utils/files/getAndOpenFile";
-import {getProfileColorByName} from "@/services/local/default-personalization";
+import { getProfileColorByName } from "@/services/local/default-personalization";
 import { getChatMessages, sendMessageInChat, getChatRecipients } from "@/services/chats";
```

Comment:
pas nécessaire mais je propose quand même mdr

```suggestion
import {
  getChatMessages,
  sendMessageInChat,
  getChatRecipients,
} from "@/services/chats";
```